proposals/relaxed-simd/entropy.md

```diff
@@ -0,0 +1,57 @@
+# Entropy, Compat and Usage patterns
+
+The Relaxed SIMD proposal adds new operations that take advantage of the underlying hardware for accelerated performance by loosening the portability constraints of fixed-width SIMD, or adding new instructions that introduce local non-determinisim. This introduces a few potential risks specifically for browser engine implementations, though it’s possible that one or more of these might be applicable to other environments: 
+* Identifying underlying characteristics of the device (processor information if the engine that implements this proposal used the most optimal lowerings)
+* Possibly identifying information about the browser currently in use (i.e. if a sufficiently motivated user writes a hand tuned Wasm function when engines use different lowerings for instructions)
+* Compatibility risks, i.e. if code compiled for one browser works differently on a different browser (This is specific to observable behavior changes, and not performance differences)
+
+This document is an attempt to collate information that is already available in the issues filed for each of the operations on fingerprinting risk, quantify the risk for fingerprinting/compat issues and provide some information about usage patterns. 
+
+## Instruction summary
+
+* **relaxed i8x16.swizzle**<br>
+  *Entropy exposed:* Differences between Intel/ARM/RISC-V<br>
```

Comment:
Lets change to x86-64/ARM64. x86-64 isn't just Intel, and RISC-V isn't really relevant yet.

Reply:
I think we are support 32-bit x86, though that is becoming exceedingly rare. With 32-bit Arm denormals were the reasons why we can't support, if I remember correctly.

Reply:
We support 32-bit x86 and ARM, but they are less relevant.

Reply:
I'll edit to remove RISC-V. While the 32-bit may be less relevant, I would assume we still want to include them for the purpose of this analysis to make sure that we're not either making them more identifiable, or have compat bugs on older hardware that are hard to reproduce. Agree that x86-64 isn't just Intel, but what granularity makes sense to surface here? Perhaps explicitly calling out x86-64, ia-32?

Reply:
> We support 32-bit x86 and ARM, but they are less relevant.

Somebody brought up 32-bit Arm not having denormals in discussion about deterministic FMA, denormals required by the standard. This might be a nit, I think it is OK to include it.

> Agree that x86-64 isn't just Intel, but what granularity makes sense to surface here? Perhaps explicitly calling out x86-64, ia-32?

Probably just x86 and Arm, it is reasonable to assume that means both 32-bit and 64-bit.